rest/model/src/main/java/io/syndesis/model/Kind.java

```diff
@@ -24,30 +24,63 @@
 
 public enum Kind {
 
-    Action(io.syndesis.model.connection.Action.class),
-    Connection(io.syndesis.model.connection.Connection.class),
-    Connector(io.syndesis.model.connection.Connector.class),
-    ConnectorGroup(io.syndesis.model.connection.ConnectorGroup.class),
-
-    Environment(io.syndesis.model.environment.Environment.class),
-    EnvironmentType(io.syndesis.model.environment.EnvironmentType.class),
-    Extension(io.syndesis.model.extension.Extension.class),
-    Organization(io.syndesis.model.environment.Organization.class),
-
-    Integration(io.syndesis.model.integration.Integration.class),
-    IntegrationRuntime(io.syndesis.model.integration.IntegrationRuntime.class),
-    Step(io.syndesis.model.integration.Step.class),
-
-    Permission(io.syndesis.model.user.Permission.class),
-    Role(io.syndesis.model.user.Role.class),
-    User(io.syndesis.model.user.User.class),
-
-    TechExtension(io.syndesis.model.techextension.TechExtension.class),
-    TechExtensionAction(io.syndesis.model.techextension.TechExtensionAction.class),
+    Action(
+        io.syndesis.model.connection.Action.class,
+        io.syndesis.model.connection.Action.VERSION),
```

Comment:
How does this translate to deployment.json where we have

```yaml
   ...
  kind: "Action"
  ....
```

I'm really not sure whether we should put the version into the kind.